bftengine/src/preprocessor/tests/messages/PreProcessResultMsg_test.cpp

```diff
@@ -30,6 +30,7 @@ bftEngine::ReplicaConfig& createReplicaConfigWithExtClient(uint16_t fVal, uint16
 
   auto clientPrincipalIds = std::set<uint16_t>{uint16_t(config.numReplicas + 1)};
   config.publicKeysOfClients.insert(make_pair(config.publicKeysOfReplicas.begin()->second, clientPrincipalIds));
+  config.clientTransactionSigningEnabled = true;
```

Comment:
Does the test for  the case
`config.clientTransactionSigningEnabled = false
`
Is completely irrelevant now?
If it is still relevant now, I would recommend having here a parametrized test - run all tests or some parts with thsi option set to true and false.

Reply:
added tests for tx_signing disabled also.